lib/grammers-client/src/types/message.rs

```diff
@@ -504,6 +504,17 @@ impl Message {
         self.client.send_message(&self.chat(), message).await
     }
 
+    /// Respond to this message by sending a album in the same chat, but without directly
+    /// replying to it.
+    ///
+    /// Shorthand for `Client::send_album`.
+    pub async fn respond_album(
+        &self,
+        medias: Vec<InputMedia>,
```

Comment:
I had in mind a slice of `InputMedia`. But perhaps using a `Vec` is okay, as it means we can take ownership and reduce clones. Not sure what's best.

Reply:
I think `Vec` is the best option right now

Reply:
Not entirely convinced. But we can definitely merge it like this for the time being and see how it feels.

Reply:
Ok.